metricbeat/module/kubernetes/_meta/config.yml

```diff
@@ -21,3 +21,10 @@
     - state_container
   period: 10s
   hosts: ["kube-state-metrics:8080"]
+
+# Kubernetes events
+- module: kubernetes
+  metricsets:
+    - event
+  kube_config: ${HOME}/.kube/config
+  in_cluster: false
```

Comment:
perhaps we should leave `in_cluster: true` as default? I guess it will be the most common case